### PR TITLE
Add the Docker Usage in README, Remove the create release section in Github Actions

### PR DESCRIPTION
**Pull request by harryyikhchan**

### CODE REVIEW

File: .github/workflows/main.yml
Comment (christopheralex) [inline .github/workflows/main.yml]: Why is this block being removed ? Isn't this how we did github actions defined by tiffany . So everytime a docker is updated and pushed a release for the docker is created and pushed to dockerhub ?

Comment (harryyikhchan) [reply .github/workflows/main.yml]: When we push a new commit to main branch, it will create a new release for us. We may end up having dozens of releases (v0.x.0) if we keep committing many times. We will create a release (v1.0.0) anyway. So I thought we don't need this block.

Comment (christopheralex) [reply .github/workflows/main.yml]: Oh it only creates a new release when there is a new change in the docker file in that commit . Else nothing happens. Like now we have only one release which is when we added the dockerfile 8 hours ago .

Comment (christopheralex) [reply .github/workflows/main.yml]: As per my understanding docker release and your code releases are different .

**Comment by christopheralex** [comment]
This is under review . Please don't merge .

File: .github/workflows/main.yml
Comment (harryyikhchan) [reply .github/workflows/main.yml]: Yes, they are under different steps but using the same tag.
Github release is under actions/create-release@v1
Docker release is under elgohr/Publish-Docker-Github-Action@master

But you are right. It only creates a new release when dockerfile is changed. Let me revert the change

Comment (christopheralex) [reply .github/workflows/main.yml]: Cool 👍 once you revert the change you can merge this . Thanks harry.

**Review by christopheralex — APPROVED** [review]
Approved

**Pull request merged**
